.pre-commit-config.yaml

```diff
@@ -1,5 +1,5 @@
 default_language_version:
-  python: python3.8
+  python: python3
```

Comment:
This change is to ensure we can run pre-commit across any Python3 environment. Fedora 33 has python 3.9 as the default.